### DISCUSSION TITLE
Windows protected your PC - SmartScreen notification during install

### DISCUSSION THREAD

**Discussion by arnisjuraga** [d2phap/ImageGlass#764] [General] [answered]
### System information:
 - Windows OS version: Windows 10
 - ImageGlass version: 7.6.4.30_x64

### To Reproduce
1. Download
2. Install

Is it "OK" and save to install anyway? 

![image](https://user-images.githubusercontent.com/2185289/80689662-51e7c680-8ad6-11ea-8397-28fd0271fade.png)

**Comment by d2phap** [comment, marked as answer]
Hi @arnisjuraga 

This is because the executable files are not signed (mentioned at #97). 
Alternately, you can check and compare the hash code SHA-1 provided on ImageGlass website to make sure the downloaded files are original.

Thank you
